### ISSUE TITLE
Exoplayer version 2.13.1 not avaiable on repos.

### ISSUE THREAD

**Issue by NelsonJTSM** [ryanheise/just_audio#495]
**Which API doesn't behave as documented, and how does it misbehave?**
The just_audio example code should run on the Android emulator. When you try to run using ```flutter run```, gradle cannot find exoplayer version 2.13.1 in any of the available repos. 

I've tried to upgrade exoplayer to the latest version (1.15.0) on a fork of the project, however some functions used by just_audio are deprecated in the latest version of exoplayer.

**Minimal reproduction project**
The example

**To Reproduce (i.e. user steps, not code)**
```sh
git clone https://github.com/ryanheise/just_audio.git
cd just_audio/just_audio/example
flutter pub get
flutter run
```

**Error messages**

```
Using hardware rendering with device Android SDK built for x86. If you notice
graphics artifacts, consider enabling software rendering with
"--enable-software-rendering".
Launching lib/main.dart on Android SDK built for x86 in debug mode...
                                                                        
FAILURE: Build failed with an exception.                                
                                                                        
* What went wrong:                                                      
Execution failed for task ':app:mergeDebugAssets'.                      
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.google.android.exoplayer:exoplayer-core:2.13.1. 
     Searched in the following locations:                               
       - https://dl.google.com/dl/android/maven2/com/google/android/exoplayer/exoplayer-core/2.13.1/exoplayer-core-2.13.1.pom
       - https://repo.maven.apache.org/maven2/com/google/android/exoplayer/exoplayer-core/2.13.1/exoplayer-core-2.13.1.pom
       - https://storage.googleapis.com/download.flutter.io/com/google/android/exoplayer/exoplayer-core/2.13.1/exoplayer-core-2.13.1.pom
     Required by:                                                       
         project :app > project :just_audio                             
   > Could not find com.google.android.exoplayer:exoplayer-dash:2.13.1. 
     Searched in the following locations:                               
       - https://dl.google.com/dl/android/maven2/com/google/android/exoplayer/exoplayer-dash/2.13.1/exoplayer-dash-2.13.1.pom
       - https://repo.maven.apache.org/maven2/com/google/android/exoplayer/exoplayer-dash/2.13.1/exoplayer-dash-2.13.1.pom
       - https://storage.googleapis.com/download.flutter.io/com/google/android/exoplayer/exoplayer-dash/2.13.1/exoplayer-dash-2.13.1.pom
     Required by:                                                       
         project :app > project :just_audio                             
   > Could not find com.google.android.exoplayer:exoplayer-hls:2.13.1.  
     Searched in the following locations:                               
       - https://dl.google.com/dl/android/maven2/com/google/android/exoplayer/exoplayer-hls/2.13.1/exoplayer-hls-2.13.1.pom
       - https://repo.maven.apache.org/maven2/com/google/android/exoplayer/exoplayer-hls/2.13.1/exoplayer-hls-2.13.1.pom
       - https://storage.googleapis.com/download.flutter.io/com/google/android/exoplayer/exoplayer-hls/2.13.1/exoplayer-hls-2.13.1.pom
     Required by:                                                       
         project :app > project :just_audio                             
   > Could not find com.google.android.exoplayer:exoplayer-smoothstreaming:2.13.1.
     Searched in the following locations:                               
       - https://dl.google.com/dl/android/maven2/com/google/android/exoplayer/exoplayer-smoothstreaming/2.13.1/exoplayer-smoothstreaming-2.13.1.pom
       - https://repo.maven.apache.org/maven2/com/google/android/exoplayer/exoplayer-smoothstreaming/2.13.1/exoplayer-smoothstreaming-2.13.1.pom
       - https://storage.googleapis.com/download.flutter.io/com/google/android/exoplayer/exoplayer-smoothstreaming/2.13.1/exoplayer-smoothstreaming-2.13.1.pom
     Required by:                                                       
         project :app > project :just_audio                             
                                                                        
* Try:                                                                  
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
                                                                        
* Get more help at https://help.gradle.org                              
                                                                        
BUILD FAILED in 19s                                                     
Running Gradle task 'assembleDebug'...                                  
Running Gradle task 'assembleDebug'... Done                        20.9s
Exception: Gradle task assembleDebug failed with exit code 1
```

**Expected behavior**
The just_audio example code should run using the Android emulator.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: MacOS + 11.4

**Smartphone (please complete the following information):**
 - Device: Android Emulator
 - OS: Android 11.0

**Flutter SDK version**
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.2.3, on macOS 11.4 20F71 darwin-x64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.59.1)
[✓] Connected device (2 available)

• No issues found!
```

**Additional context**
N/A

**Comment by ryanheise** [comment]
Just copying my responses over from your other issue:

> Hmm, it must have been removed recently. I'll update the dependency and migrate the deprecated code.

> I'm not sure why, but I can't seem to reproduce the error. I've done a flutter clean and also deleted my gradle cache, but it still seems to resolve the ExoPlayer dependency.

**Comment by ryanheise** [comment]
Upgrading to 2.15.0 has some wider implications which I will put off for now. It will be a breaking change and hence go into the 0.10.0 release.

For now, it is instead possible to upgrade to 2.13.3. Can you try the fix on the `fix/exoplayer_deps` branch?

**Comment by NelsonJTSM** [comment]
The change in ```fix/exoplayer_deps``` worked as expected.

**Comment by ryanheise** [comment]
Fixed in release 0.9.8.
